ts/components/services/ServicePreferenceSummary.tsx

```diff
@@ -0,0 +1,57 @@
+import { View } from "native-base";
+import * as React from "react";
+import { connect } from "react-redux";
+import { fromNullable } from "fp-ts/lib/Option";
+import { InfoBox } from "../box/InfoBox";
+import { IOStyles } from "../core/variables/IOStyles";
+import { IOColors } from "../core/variables/IOColors";
+import { H5 } from "../core/typography/H5";
+import I18n from "../../i18n";
+import { Link } from "../core/typography/Link";
+import { GlobalState } from "../../store/reducers/types";
+import { Dispatch } from "../../store/actions/types";
+import { navigateToServicePreferenceScreen } from "../../store/actions/navigation";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+const mapChoiceLabel = new Map<string, string>([
+  ["AUTO", I18n.t("services.optIn.preferences.quickConfig.value")],
+  ["MANUAL", I18n.t("services.optIn.preferences.manualConfig.value")]
+]);
+
+const getChoiceLabel = (status: string) =>
+  fromNullable(mapChoiceLabel.get(status)).getOrElse(
+    I18n.t("services.optIn.preferences.unavailable")
```

Comment:
```suggestion
    mapChoiceLabel.get(status) ??
  I18n.t("services.optIn.preferences.unavailable")
```
It is pretty the same but I find the nullish coalescing operator more readable and I guess passing throught an `Option` is less efficient in term of memory consumption
it is a just a question of syntactic sugar. What do you think?